requirements/test.txt

```diff
@@ -4,11 +4,6 @@
 # sha256: ZdKqaLKOfTEjO7K6jrMc2kDkZx-KwtayQeNYyWUqdLk
 apipkg==1.4
 
-# sha256: gqJTJFrtmHXxQd2Yp0YlcrypPQZM8xF70z3kth7hjmg
-# sha256: LCZCVPbPzmTDvZ1IiFIICTpzuQlWRfYA3gonfOAeoOU
-# sha256: h9QBPQYl1HiaT1a415oE1c5tsRUrtl8dOXRPdwmjZrQ
-beautifulsoup4==4.4.1
```

Comment:
Given that the tests pass with the older version of BeautifulSoup in requirements/base.txt, and that's the version being used in production now, the safest thing to do is to merge this as-is and update the version in requirements/base.txt in a later PR.